application/classes/Ushahidi/Core.php

```diff
@@ -672,6 +677,22 @@ public static function init()
 		$di->setter['Ushahidi_Listener_PostListener']['setWebhookRepo'] =
 			$di->lazyGet('repository.webhook');
 
+		// Add Intercom Listener to Config
```

Comment:
It might be too late for this.. but could we move these into src/App ? (fine if not, I'll move them in the lumen branch instead)